presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -239,6 +242,23 @@ public static void setReadColumns(Configuration configuration, List<Integer> rea
         configuration.setBoolean(READ_ALL_COLUMNS, false);
     }
 
+    public static Optional<CompressionCodec> getCompressionCodec(TextInputFormat inputFormat, Path file)
+            throws NoSuchFieldException, IllegalAccessException
+    {
+        if (inputFormat == null || file == null) {
```

Comment:
Remove these checks, since we should never call the method with null. (As a general code style, we don't write code to be lenient of nulls in Presto.)

Reply:
Removed the checks

Reply:
Looks like the check is still here?